lib/web_ui/test/ui/line_metrics_test.dart

```diff
@@ -176,5 +176,5 @@ Future<void> testMain() async {
 
     // In Roboto, the width should be 11 here. In the test font, it would be square (16 points)
     expect(metrics!.width, 11);
-  }, solo: true);
```

Comment:
We desperately need a lint that prevents this.

Reply:
Actually, `solo` is marked with `@Deprecated`. We should not have been able to check it in 🤔

Reply:
Yeah I was wondering about this. We used to have a lint I thought, but it disappeared apparently. I was surprised to find I had been able to check this in too